### ISSUE TITLE
Pipeline shows an error  in web console for unstable jenkins builds

### ISSUE THREAD

**Issue by employe1** [openshift/origin#21561]
Hello,

In our jenkins pipeline we are doing some security tests which outputs some unstable results and pipeline ends as unstable. This dosen't effect anything in the pod creation but the Ui is pointing only the error state and eveything is greyed out. can I have green pipleline during unstable builds in jenkins.
![errorpipeline](https://user-images.githubusercontent.com/42376930/49146348-418c5480-f302-11e8-8a23-b72304e66757.PNG)

using Version 3.9

To sum it up: "Jenkins build result is unstable but shows up as failed in the Openshift Pipeline View"

**Comment by ilopezv** [comment]
same problem as me on OCP 3.11
